test/ui-testing/CustomInsert.test.js

```diff
@@ -0,0 +1,61 @@
+const timeout = process.env.SLOWMO ? 60000 : 10000;
+const fs = require('fs');
+beforeAll(async () => {
+	path = fs.realpathSync('file://../examples/index.html');
+	await page.goto('file://' + path, {waitUntil: 'domcontentloaded'});
+});
+
+describe('Custom Insert text', () => {
+	test('Add Custom Insert text in rich text mode', async () => {
+	await page.waitForSelector('.ple-module-body');
+	// Click on insert text button
+	await page.click('.woofmark-command-insert');
+	// Select the option 'Nodes' from Menu: What Do you want to insert?
+	await page.waitForSelector('#dropdownMenu1');
+	await page.click('#dropdownMenu1');
+	await page.waitForSelector('#Nodes');
```

Comment:
You can use page.select() for dropdown list.

Reply:
I have not used <select> tag so can't use page.select()